x-pack/test/fleet_api_integration/apis/epm/data_stream.ts

```diff
@@ -14,18 +14,13 @@ import { setupFleetAndAgents } from '../agents/services';
 
```

Comment:
A lot of indentation changes in this file because I had to add a second `describe` block. Best viewed with whitespace disabled in GitHub.